app/controllers/users/phone_confirmation_controller.rb

```diff
@@ -33,11 +33,12 @@ def assign_phone
       old_phone = current_user.phone
       @updating_existing_number = old_phone.present?
       if @updating_existing_number
-        analytics.track_event('User changed and confirmed their phone number')
+        analytics.track_event('User changed their phone number')
         SmsSenderNumberChangeJob.perform_later(old_phone)
```

Comment:
But their verified number doesn't actually change until proofing is complete, right?

I would say we should notify the user every time they re-proof perhaps, but the phone number is not any more important that any other fields in the `profiles` table, right?

The reason we notify the user about OTP phone number change is because it effects account security directly.